docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -4,9 +4,19 @@ description: This guide explains how to setup a Slack plugin for Teleport for pr
 h1: Teleport Slack Plugin Setup
 ---
 
-This guide will talk through how to set up Teleport with Slack. Teleport's Slack integration notifies individuals and channels of Access Requests.
+This guide will explain how to set up Teleport with Slack. Teleport's Slack
+integration notifies individuals and channels of access requests.
 
-#### Example Slack Request
+<Notice
+  type="danger"
+  scope="oss"
+>
+  This guide requires a commercial edition of Teleport. The open source
```

Comment:
Is commercial and enterprise the same thing?

Reply:
In some docs guides we have used "commercial" to refer to "Cloud or Enterprise." I've made this a bit more specific here.